packages/playground/src/components/filters/TfFiltersContainer.vue

```diff
@@ -68,7 +68,7 @@
     >
       <div>
         <VExpandTransition mode="in-out">
-          <VForm :disabled="loading" v-show="!collapsible || filterOpened">
+          <VForm :disabled="loading" v-show="!collapsible || filterOpened" @keyup.enter="apply">
```

Comment:
if we focus on any input in form and click enter without updating any input it will just run `apply` which is not required at all

Reply:
fixed.